packages/bs-reform/src/ReFormNext.re

```diff
@@ -16,7 +16,7 @@ module Make = (Config: Config) => {
   module Validation = {
     type t =
       | Email(Config.field(string)): t
-      | Optional(Config.field('a)): t
+      | StringNonEmpty(Config.field(string)): t
```

Comment:
actually Optional should stay! Optional is for fields that don't require any validation. Maybe we should rename it?

Reply:
I will keep Optional validator. But about the name, maybe it needs to be renamed, do you have any ideas?

Reply:
`NoValidation`

Reply:
or `Any`

Reply:
NoValidation is nice